.travis.yml

```diff
@@ -18,9 +19,9 @@ before_script:
   - sleep 5
 script:
   - npm run lint
-  - npm run unit
+  - if [[ $(node -v) =~ ^v4.* ]]; then npm run unit:harmony; else npm run unit; fi
```

Comment:
hmmm actually i'm seeing a pattern here using `matrix` that seems a bit smarter: https://github.com/arrizalamin/js-function-reflector/blob/72cf820139cac6f4ce75a8ef1d1efaf01e2accf4/.travis.yml